src/auto-trace.helper.js

```diff
@@ -84,14 +74,26 @@ export function appendExtraContext(error, extraContext){
 	return errOut
 }
 
+//This function creates an error and by default removes two frames from the error stack (to remove this method, and the caller, which will be a function inside auto-trace)
+//Optionaly extraframesToRemove can be specified to remove more frames if the stack trace will contain more auto-trace refrences.
+export function createError(extraframesToRemove = 1){
+	const error = new Error();
+	//We remove an extra frame since this function will create a frame as well
+	const newStack = error.stack.split('\n');
+	if (newStack.length > extraframesToRemove+1) {
+		newStack.splice(1,extraframesToRemove+1);
```

Comment:
Are we starting at 1 because the 0th line of stacktraces is the message and we don't want to get rid of that?

Reply:
Correct, I can add a comment to reflect that